website/content/docs/nia/compatibility.mdx

```diff
@@ -13,10 +13,12 @@ The following tables list the Consul-Terraform-Sync (CTS) version compatibility
 
 Below are CTS versions with supported Consul versions. The latest CTS binary supports the three most recent Consul minor versions, along with their latest patch versions.
 
-| CTS Version         | Consul OSS & Enterprise Version |
-| :------------------ | :------------------------------ |
-| CTS Enterprise 0.3+ | 1.8+, HCP-Consul                |
-| CTS OSS 0.1+        | 1.8+, HCP-Consul                |
+| CTS Version         | Consul OSS & Enterprise Version | HCP Consul Version |
+| :------------------ | :------------------------------ | :----------------- |
+| CTS Enterprise 0.6+ | 1.8+                            | 1.9+               |
+| CTS Enterprise 0.3+ | 1.8+                            | N/A                |
```

Comment:
Nit: wondering if it would make sense to update `CTS Enterprise 0.3+` to something like`CTS Enterprise 0.3-0.5`? When I initially read it, it seemed as though 0.6 could fall in the category of 0.3+. But, this might just be me!

Reply:
Oh, that's a good point! I'll update it here and for CTS OSS.